docs/content/docs/3.guide/3.using-healthchecks-with-laravel.md

```diff
@@ -0,0 +1,118 @@
+---
+head.title: 'Native Docker Health Checks for Laravel - Docker PHP - Server Side Up'
+description: 'Stop guessing if Laravel is up and running. Our health checks have you covered to ensure Laravel is running properly. It supports HTTP checks, Laravel Horizon, Reverb, Scheduler, and Queue.'
+layout: docs
+---
+
+# Native Docker Health Checks for Laravel
+::lead-p
+Dialing in health checks are very important for ensuring your application is running smoothly and that you're able to deploy updates without any downtime. This guide will explain how our health checks work and how you can use them to your advantage.
+::
+
+## What are Health Checks?
+
+Health checks are a way to check the status of your application. Whenever a container is started, a health check is performed. If the health check fails, the container will be restarted or marked as unhealthy. Health checks are very important for rolling updates and ensuring your application can start up in order if you have services that depend on each other.
+
+## Our Health Checks
+We offer a number of health check commands, specifically for Laravel. You can find these commands are prefixed with `healthcheck-` and are located in [`/usr/local/bin`](https://github.com/serversideup/docker-php/tree/main/src/common/usr/local/bin). The examples below show how to use these health checks in your `docker-compose.yml` file, but you can also use them in other environments.
+
+## Default Health Check Settings
+By default, our Dockerfiles ship with the following health check commands:
+
+| Variation | Health Check Command |
+| --------- | -------------------- |
+| `cli`     | (none)               |
+| `fpm`     | [`php-fpm-healthcheck`](https://github.com/renatomefi/php-fpm-healthcheck) |
+| `fpm-apache`     | `curl --insecure --silent --location --show-error --fail http://localhost:8080$HEALTHCHECK_PATH` |
+| `fpm-nginx` | `curl --insecure --silent --location --show-error --fail http://localhost:8080$HEALTHCHECK_PATH` |
+
+::note
+Notice how `fpm-apache` and `fpm-nginx` have a `$HEALTHCHECK_PATH` variable? This is because you can specify a path for the health check to validate.
+::
+
+## Changing the Health Check Path
+By default, you can see our [Environment Variable Specification](/docs/reference/environment-variable-specification) shows the `HEALTHCHECK_PATH` variabe is set to `/healthcheck`.
```

Comment:
@jaydrogers Here is a typo: variabe

Reply:
Fixed in https://github.com/serversideup/docker-php/commit/468287a6e90124d9368f2b337f3f12edbacc7811

Thanks @szepeviktor 🤝